homeassistant/components/switch/rest.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+"""
+homeassistant.components.switch.rest
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Allows to configure a REST switch.
+
+Configuration:
+
+switch:
+  platform: rest
+  name: "Bedroom Switch"
+  resource: "http://IP_ADDRESS/ENDPOINT"
+  body_on: "ON"
+  body_off: "OFF"
```

Comment:
I agree that the context is more important then the underlying library, for all we know the implementation can change but the configuration should not need to be changed.